go.mod

```diff
@@ -49,26 +49,26 @@ require (
 	github.com/phayes/freeport v0.0.0-20180830031419-95f893ade6f2
 	github.com/pingcap/check v0.0.0-20211026125417-57bd13f7b5f0
 	github.com/pingcap/errors v0.11.5-0.20211224045212-9687c2b0f87c
-	github.com/pingcap/failpoint v0.0.0-20220303073211-00fea37feb66
-	github.com/pingcap/kvproto v0.0.0-20220328072018-6e75c12dbd73
-	github.com/pingcap/log v0.0.0-20211215031037-e024ba4eb0ee
-	github.com/pingcap/tidb v1.1.0-beta.0.20220412180037-d07b66ea638c
-	github.com/pingcap/tidb-tools v6.0.0-alpha.0.20220317013353-dfc5146f4746+incompatible
-	github.com/pingcap/tidb/parser v0.0.0-20220412180037-d07b66ea638c
-	github.com/prometheus/client_golang v1.11.0
+	github.com/pingcap/failpoint v0.0.0-20220423142525-ae43b7f4e5c3
+	github.com/pingcap/kvproto v0.0.0-20220517085838-12e2f5a9d167
+	github.com/pingcap/log v1.1.0
+	github.com/pingcap/tidb v1.1.0-beta.0.20220528045048-5495dc6c4360
+	github.com/pingcap/tidb-tools v6.0.1-0.20220516050036-b3ea358e374a+incompatible
+	github.com/pingcap/tidb/parser v0.0.0-20220528045048-5495dc6c4360
+	github.com/prometheus/client_golang v1.12.2
```

Comment:
Conflicts to https://github.com/pingcap/tiflow/issues/5494 ?

Reply:
Need check：
1. Check If we have used following three metric.  If true, need modify the grafna rules in this pr.
go_gc_heap_allocs_by_size_bytes_total -> go_gc_heap_allocs_by_size_bytes,
go_gc_heap_frees_by_size_bytes_total -> go_gc_heap_allocs_by_size_bytes
go_gc_pauses_seconds_total -> go_gc_pauses_seconds.

2. Check if we use NewGoCollection(). If true, add  `GoRuntimeMemStatsCollection | GoRuntimeMetricsCollection`  in this pr.